.pre-commit-config_template.yaml

```diff
@@ -41,6 +41,8 @@ repos:
     - --config=nightly_ruff.toml
     args:ci:
     - --config=nightly_ruff.toml
+    args:docker_autoupdate:
+    - --config=nightly_ruff.toml
```

Comment:
there isnt a way to combine these 3 cases?

Reply:
I tried searching online but couldn't find a way to combine them